moveit_ros/visualization/src/moveit_ros_visualization/moveitjoy_module.py

```diff
@@ -169,6 +169,62 @@ def __init__(self, msg):
         self.orig_msg = msg
 
 
+class PS3DualShockStatus(JoyStatus):
+    def __init__(self, msg):
+        JoyStatus.__init__(self)
+        # creating from sensor_msgs/Joy
+        if msg.buttons[0] == 1:
+            self.cross = True
+        else:
+            self.cross = False
```

Comment:
Personally, I would prefer the more compact format:
```suggestion
        self.cross = (msg.buttons[0] == 1)
```
However, that could be part of a separate cleanup commit.